README.md

```diff
@@ -292,6 +292,37 @@ The metadata should be written directly to the request body.
 }
 ```
 
+### POST /git/fetch
+
+Does a git fetch on the repository (relevant for when runners clone down git repositories). Expects an empty json body.
+
+### GET /git/branches
+
+Returns an array of branches with their `name`, `commit_hash` (where branch pointer is), `time` (of last commit) and `message` (of last commit in a string array split with respect to newline characters)
+
+#### Response
+
+```
+{
+  "status": "success",
+  "data": [
+    {
+      "name": "main",
+      "commit_hash": "ede307e23b2137ba2c7c3270e52f354f224942af",
+      "time": 1722436575,
```

Comment:
I find RFC3339 timestamps nicer in a JSON context, but that's a matter of personal preference I guess.

Do we have precedent for using one or the other in our files?

Reply:
Ok I looked it up and orderly's packet metadata uses Unix timestamps already. Bit of a shame IMO but it makes sense to stay consistent then.